internal/services/costmanagement/export_resource_base.go

```diff
@@ -100,10 +108,17 @@ func (br costManagementExportBaseResource) arguments(fields map[string]*pluginsd
 							string(exports.TimeframeTypeTheLastMonth),
 							string(exports.TimeframeTypeWeekToDate),
 							string(exports.TimeframeTypeMonthToDate),
+							string(exports.TimeframeTypeMonthToDate),
 							// TODO Use value from SDK after https://github.com/Azure/azure-rest-api-specs/issues/23707 is fixed
 							"TheLast7Days",
 						}, false),
 					},
+
+					"data_version": {
```

Comment:
Is there some validation we could add for this property? Even just a `validation.StringIsNotEmpty`?

Reply:
Yeah, that's the best we can do unfortunately. It's usually of the form yyyy-mm-dd, but then with some optional -preview or -internal suffixes. It doesn't feel right to constrain to those values since they don't seem structured other than "might contain a date"